arch/arm64/core/Kconfig

```diff
@@ -143,15 +143,6 @@ config ARM64_SAFE_EXCEPTION_STACK
 	  used for user stack overflow checking, because kernel stack support
 	  the checking work.
 
-config ARM64_ENABLE_FRAME_POINTER
-	bool
-	default y
-	depends on OVERRIDE_FRAME_POINTER_DEFAULT && !OMIT_FRAME_POINTER
-	help
-	  Hidden option to simplify access to OVERRIDE_FRAME_POINTER_DEFAULT
-	  and OMIT_FRAME_POINTER. It is automatically enabled when the frame
-	  pointer unwinding is enabled.
```

Comment:
I would keep this one but deprecate it. Remember to select deprecated, add deprecated to the prompt, and mention the preferred option in the help string.

If the riscv option was not present in 3.6.0, there is no need to deprecate it.

Reply:
Fixed